.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: CI
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: "30 4 * * 4"
+
+defaults:
+  run:
+    working-directory: 'ansible-role-candlepin'
+
+jobs:
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
with github actions our options are ubuntu/windows/osx or run your own;  we don't have external hardware to run our own...